docs/sources/reference/playlist.md

```diff
@@ -11,59 +11,130 @@ weight = 4
 
 # Playlist
 
-The Playlist is a special type of Dashboard that rotates through a list of Dashboards. A Playlist can be a great way to build situational awareness, or just show off your metrics to your team or visitors.
+## About Playlists
```

Comment:
```suggestion
```

From the style guide:

> Avoid following one heading with another heading.

Reply:
Ah, I considered this! But figured that I was writing this to eventually be broken up into further submenu options under Playlists. Like:
Dashboards
- Playlists
-- About playlists
-- Create a playlist

Where clicking Playlists opens a dropdown to select a topic. 

Until that happens, if it does, I'll nix About playlists and put that section under the main Playlists heading. Thoughts @oddlittlebird ?